modules/serverless-install-eventing-yaml.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-eventing.adoc
+
+[id="serverless-install-eventing-yaml_{context}"]
+= Installing Knative Eventing using YAML
+
+.Procedure
+
+. Create a file named `eventing.yaml`.
+. Copy the following sample YAML into `eventing.yaml`:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+    name: knative-eventing
+    namespace: knative-eventing
+----
+. Apply the `eventing.yaml` file:
++
+----
+$ oc apply -f eventing.yaml
+----
+
+.Verification steps
+
+. To verify the installation is complete, enter the following command:
```

Comment:
Also, for web console we instruct users how to check what knative eventing resources are created, and where to check for their status. For analogy it should be noted here as well. Like:

> Call followaing command to see internal knative-eventing resources being created:
>
> `oc get pods -n knative-eventing`

Reply:
Sure. On it.

Reply:
I took output from Serverless 1.7:

Eventing:

```
$ oc get pods -n knative-eventing
NAME                                   READY   STATUS    RESTARTS   AGE
broker-controller-58765d9d49-g9zp6     1/1     Running   0          7m21s
eventing-controller-65fdd66b54-jw7bh   1/1     Running   0          7m31s
eventing-webhook-57fd74b5bd-kvhlz      1/1     Running   0          7m31s
imc-controller-5b75d458fc-ptvm2        1/1     Running   0          7m19s
imc-dispatcher-64f6d5fccb-kkc4c        1/1     Running   0          7m18s
```

Serving:

```
$ oc get pods -n knative-serving
NAME                               READY   STATUS    RESTARTS   AGE
activator-5c596cf8d6-5l86c         1/1     Running   0          9m37s
activator-5c596cf8d6-gkn5k         1/1     Running   0          9m22s
autoscaler-5854f586f6-gj597        1/1     Running   0          9m36s
autoscaler-hpa-78665569b8-qmlmn    1/1     Running   0          9m26s
autoscaler-hpa-78665569b8-tqwvw    1/1     Running   0          9m26s
controller-7fd5655f49-9gxz5        1/1     Running   0          9m32s
controller-7fd5655f49-pncv5        1/1     Running   0          9m14s
kn-cli-downloads-8c65d4cbf-mt4t7   1/1     Running   0          9m42s
webhook-5c7d878c7c-n267j           1/1     Running   0          9m35s
```